packages/fxa-settings/src/models/contexts/NimbusContext.ts

```diff
@@ -60,7 +61,8 @@ export interface NimbusProviderProps {
 }
 
 export function NimbusProvider({ children }: NimbusProviderProps) {
-  const { config, uniqueUserId } = useContext(AppContext);
+  const { config, uniqueUserId} = useContext(AppContext);
```

Comment:
Nit, I think something might be off with your prettier settings... we shouldn't need the diff here or on L47, and it'll get changed back next time someone works on this file.